.github/workflows/docs.yml

```diff
@@ -32,11 +32,11 @@ jobs:
       - name: Install dependencies
         run: >
           pip3 install --upgrade pip
-          && pip install -r docs/requirements.txt
+          && pip install -r docs/old/requirements.txt
```

Comment:
We also need to point the update-bot to the new location:
https://github.com/acts-project/acts/blob/3743675fb2839e840ceb2ad298cb46606c131414/.github/workflows/update-pip-requirements.yml#L36